src/@next/Tabs/Tabs.tsx

```diff
@@ -39,9 +39,37 @@ export const Tabs = React.forwardRef<HTMLDivElement, TabsProps>(function Tabs(
   }: TabsProps,
   ref
 ) {
+  const tabsHeaderRef = useRef(null);
   const tabLength = tabs.length;
 
   const [selectedTabIndex, setSelectedTabIndex] = useState(0);
+  const [canScrollLeft, setCanScrollLeft] = useState(false);
+  const [canScrollRight, setCanScrollRight] = useState(false);
+  const [isDragging, setIsDragging] = useState(false);
+  const [dragStartX, setDragStartX] = useState(0);
+  const [scrollLeft, setScrollLeft] = useState(0);
+
+  const handleMouseDown = (event: React.MouseEvent<HTMLDivElement>) => {
```

Comment:
@ninariccimarie When I drag the tabs using the mouse, the behavior seems to be quite jerky. Does dragging prevent the click event from firing?